### DISCUSSION TITLE
1.16.5离线登录没法多人游戏

### DISCUSSION THREAD

**Discussion by DRDPsyt** [Meloong-Git/PCL#2017] [提问] [answered]
我的UUID不管在哪个启动器都没法多人（1.16.5），但是别的版本就行。一起玩的几个朋友也都是这个问题，咋解决啊

**Comment by WForst-Breeze** [comment]
经测试与查阅资料，近期 1.16.4 及 1.16.5 均无法使用离线模式进入多人游戏。
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/f9620d50-79ae-4eec-b571-8e3f74be0648)

最简单粗暴的解决方案为：断网打开 PCL2 并启动 1.16.5，问题即可解决。然后记得再连上网，这样即可正常多人游戏。

###### 还有一种相对稳定的解决方法，可以看看[这篇博客](http://ylcao.top/2023/06/09/Minecraft%E7%A6%81%E6%AD%A2%E5%A4%9A%E4%BA%BA%E6%B8%B8%E6%88%8F%E7%A0%B4%E8%A7%A3/)。同时，这篇博客也进行了相关源代码分析

**Reply by WorldHim** [reply, marked as answer]
> 经测试与查阅资料，近期 1.16.4 及 1.16.5 均无法使用离线模式进入多人游戏。
> 
> 最简单粗暴的解决方案为：断网打开 PCL2 并启动 1.16.5，问题即可解决。然后记得再连上网，这样即可正常多人游戏。
> 
> ###### 还有一种相对稳定的解决方法，可以看看[这篇博客](http://ylcao.top/2023/06/09/Minecraft%E7%A6%81%E6%AD%A2%E5%A4%9A%E4%BA%BA%E6%B8%B8%E6%88%8F%E7%A0%B4%E8%A7%A3/)。同时，这篇博客也进行了相关源代码分析

目前已有模组解决此问题，见：
Forge & Fabric：[MCTeamPotato/MultiOfflineFix](https://github.com/MCTeamPotato/MultiOfflineFix)
Fabric：[ChickenPige0n/offline-multiplayer-fabric](https://github.com/ChickenPige0n/offline-multiplayer-fabric)
